### ISSUE TITLE
interact.py killed on second run

### ISSUE THREAD

**Issue by alelasantillan** [huggingface/transfer-learning-conv-ai#44]
I did:
git clone https://github.com/huggingface/transfer-learning-conv-ai
cd transfer-learning-conv-ai
pip install -r requirements.txt
python -m spacy download en

Then It run once as follows:
python3 interact.py

It downloaded the model and I could chat with the bot. I stop the execution with Ctrl+C and then
I did again:
python3 interact.py

to get:
INFO:pytorch_transformers.modeling_utils:loading weights file /tmp/tmpzwe_u3oz/pytorch_model.bin
INFO:pytorch_transformers.modeling_utils:Weights from pretrained model not used in OpenAIGPTLMHeadModel: ['multiple_choice_head.summary.weight', 'multiple_choice_head.summary.bias']
INFO:pytorch_transformers.tokenization_utils:Assigning <bos> to the bos_token key of the tokenizer
INFO:pytorch_transformers.tokenization_utils:Assigning <eos> to the eos_token key of the tokenizer
INFO:pytorch_transformers.tokenization_utils:Assigning <pad> to the pad_token key of the tokenizer
INFO:pytorch_transformers.tokenization_utils:Assigning ('<speaker1>', '<speaker2>') to the additional_special_tokens key of the tokenizer
INFO:interact.py:Sample a personality
INFO:/home/jovyan/work/alela/dan_chatbots/transfer-learning-conv-ai/utils.py:Load tokenized dataset from cache at ./dataset_cache_OpenAIGPTTokenizer
Killed

And every time I run
pyhton3 interact.py, the same happens. Always Killed.

Somebody can explain?

On the other hand I have no way to exit the program while into the dialogue but to do Ctrl+C

Thank you, 
Alejandro

**Comment by sshleifer** [comment]
Hi. Unfortunately I can't replicate this on my machine. What versions of pytorch-transformers and pytorch are you on?

**Comment by alelasantillan** [comment]
I was able to avoid the Killed prompt and now the 
python ./interact.py
works well by deleting dataset_cache_OpenAIGPTTokenizer from the local dir before running the command.

But if I Ctrl+C and then I run it again without erasing the file dataset_cache_OpenAIGPTTokenizer it fails.
I've tried a lot of usual commands into the dialogue prompt to exit clean, without any success.
Using Ctrl+Z works, but because the process is not canceled but paused (it continues after sending the fg command, as usually happens with any excution in linux).

Regarding your question:
pytorch-ignite==0.2.1
pytorch-transformers==1.2.0
torch==1.3.0
and torch.load('dataset_cache_OpenAIGPTTokenizer') loads perfectly well within a python3 shell.

Any idea how to finish the dialogue smoothly?
